app/Http/Controllers/Assets/AssetCheckoutController.php

```diff
@@ -66,6 +67,11 @@ public function store(AssetCheckoutRequest $request, $assetId)
                 throw new CheckoutNotAllowed('You cannot check an asset out to itself.');
             }
             $asset = $this->updateAssetLocation($asset, $target);
+            
+            $status_id ='';
```

Comment:
Why is this necessary? No asset should ever have an asset status of null or `''`